spec/services/issuing_ca_service_spec.rb

```diff
@@ -95,6 +95,17 @@
     end
   end
 
+  describe '.certificate_store_issuers' do
+    it 'allowed hosts configuration contains all certificate store issuers' do
+      CertificateStore.instance.load_certs!(dir: './config/certs')
```

Comment:
Can we add a `Rails.root.join('config/certs')` so that it doesn't matter where this gets called from?

```suggestion
      CertificateStore.instance.load_certs!(dir: Rails.root.join('config/certs'))
```